app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.HumanLearning">
-        <activity android:name=".DataCreationActivity"/>
+        <activity android:name=".ui.login.GoogleSignInActivity"></activity>
```

Comment:
est-ce que le dossier 'ui' a un but différent de 'view' ?

Reply:
That wasn't my decision, but we're not even using the code that generated this anymore so we might as well name this whatever. Should it be in the top-level like the other activities in your book?